packages/ember/docs/components/docfy-output-demo/edit-page.md

```diff
@@ -20,3 +20,35 @@ repository URL as well as the branch to edit.
   {{/if}}
 </DocfyOutput>
 ```
+
+For this to work, you need to include `repository` in your docfy-config:
+
+```js
+// in your docfy-config.js
+module.exports = {
+  repository: {
+    url: 'https://github.com/@username/repo-name',
+    editBranch: 'main',
+  },
+  ...// rest of your config
+}
+```
+
+
+## Enterprise (aka on premise) git services
+
+`page.editUrl` works for Github, Bitbucket, Gitlab and Sourcehut.
+
+For on-premise instances git solutions (i.e. on-premise Gitlab, or on-premise Bitbucket), we expose the `page.relativePath` so that you might construct your own custom editUrl:
+
+```hbs 
+<DocfyOutput @fromCurrentURL={{true}} as |page|>
+  {{#if page.relativePath}}
+    <a href=(concat "http://some-enterpise.com/browse/" page.relativePath)
```

Comment:
Note: the build breaks if I use `hbs template` instead of just `hbs`...not sure why?